infrastructure/src/main/java/org/corfudb/infrastructure/SequencerServer.java

```diff
@@ -144,6 +157,9 @@ public boolean txnCanCommit(TxResolutionInfo txData) {
                         if (v > txData.getSnapshotTimestamp()) {
                             log.debug("Rejecting request due to {} > {} on streams {}",
                                     v, txData.getSnapshotTimestamp(), streamID);
+                            //System.out.println("Rejecting request due to update-timestamp on stream");
```

Comment:
Same, also, why are you using system.out.println instead of log. Please use log in the future.